app/src/main/java/com/omkarmoghe/pokemap/models/map/LatLng.java

```diff
@@ -0,0 +1,119 @@
+package com.omkarmoghe.pokemap.models.map;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Created by chris on 7/24/2016.
+ */
+
+public class LatLng implements Parcelable {
```

Comment:
name conflict with Google's LatLng class. Just adds confusion for people trying to build the project. Are we doing something in this not supported by the class provided by google?